### DISCUSSION TITLE
How to hide computer name and user name in terminal command prompt

### DISCUSSION THREAD

**Discussion by rhumbertgz** [Eugeny/tabby#7557] [Q&A] [answered]
Hello, 

In the default MacOS terminal I can hide the computer name and user name by running this `export PS1="\$ "; clear;` on the startup of the shell. 

For example: 
from `humberto@macbook  ` to `$`

How could I achieve the same in Tabby?

**Comment by Eugeny** [comment, marked as answer]
The same way, it's not terminal-specific.

**Reply by rhumbertgz** [reply]
Yes, I supposed that, but I couldn't find a setting option for that in Tabby's Preference window. Did I miss such it? or Does it have to be done manually? 
Default MacOS terminal
![Screenshot 2022-11-24 at 10 39 24](https://user-images.githubusercontent.com/10393245/203749996-eb5da906-8614-4258-9d63-7b83a73c48a7.png)

**Reply by Eugeny** [reply]
You just put these in your `~/.bashrc` / `~/.zshrc` file (depending on the shell)

**Reply by rhumbertgz** [reply]
Thanks! that worked!

**Comment by lawweiliang** [comment]
@rhumbertgz 
Please close the discussion if the question was answered. 
It would be helpful to make the discussion section clean. 

Click `mark as answered` and `close the discussion`. 😄 

![image](https://user-images.githubusercontent.com/34936929/228098333-222ac3d4-4c39-4c26-a2a8-52906647e00e.png)
